### PR TITLE
Search simple

### PR DESCRIPTION
**Pull request by f3arnil**

### CODE REVIEW

File: angular-spa-ui/js/common/modal/index.html [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
+        <div>Enter text: <b>{{ inputAdvancedSearch.text }}</b></div>
     </div>
     <div class="modal-footer">
-        <button class="btn btn-primary" type="button" ng-click="ok()">OK</button>
-        <button class="btn btn-warning" type="button" ng-click="cancel()">Cancel</button>
+        <button class="btn btn-warning" type="button" ng-click="ok()">Search</button>
+        <button class="btn btn-primary" type="button" ng-click="cancel()">Cancel</button>
     </div>
-</script>
+</script>
```
Comment (tarhunakau) [inline angular-spa-ui/js/common/modal/index.html]: \n

File: angular-spa-ui/js/search-app/search-app-ctrl.js [hunk 2]
```diff
@@ -71,4 +71,5 @@ module.exports = function (app) {
                     console.log('Error '+ error.status);
                 });
     });
+
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-app-ctrl.js]: ?

File: angular-spa-ui/js/search-app/search-app-srv.js [hunk 3]
```diff
@@ -89,4 +89,15 @@ module.exports = function (app) {
 
     });
     
+    app.service('searchStorage', function () {
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-app-srv.js:92]: ?

app.service('searchStorage', function () {
  return {
    data: {},
    params: {}
  }
});

Comment (f3arnil) [inline angular-spa-ui/js/search-app/search-app-srv.js:92]: this is storage for search module. default its empty

File: angular-spa-ui/js/search-app/search-module/search-config.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+            {name : '15 Results/page', value :'15'},
+            {name : '20 Results/page', value : '20'},
+            {name : '25 Results/page', value :'25'},
+            {name : '30 Results/page', value : '30'}
+        ],
+        searchIn: [
+                {name :'Publications', value : 'publication'},
+                {name :'Articles/Chapters', value : 'article'}
+            ]
+    }
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/search-config.js]: !

File: angular-spa-ui/js/search-app/search-module/search-ctrl.js [hunk 5]
```diff
@@ -1,58 +1,179 @@
...
+                }
+            );
+        };
+        
+        // Check has model "query" data or not - if yes returns true, else false
+        $scope.hasQuery = function () {
+            if ($scope.query) {
+                return true;
+            } else {
+                return false;
```
Comment (tarhunakau) [inline angular-spa-ui/js/search-app/search-module/search-ctrl.js]: You do not need 'else' statement this:

if (...) {
return true;
}
return false;

**Comment by tarhunakau** [comment]
Could be merged.
Code style and walls of code (angular-spa-ui/js/search-app/search-module/search-ctrl.js) must be refactored in future.

**Pull request merged**